docs/project/repository-guide.md

```diff
@@ -0,0 +1,77 @@
+    - [js-ipld](https://github.com/ipld/js-ipld): The JavaScript Implementation of IPLD.
+    - [IPLD specifications](https://github.com/ipld/specs): The set of specifications that make up IPLD.
+- [Multiformats](https://github.com/multiformats):
+    - [Multiaddr](https://github.com/multiformats/multiaddr): Composable and future-proof network addresses.
+    - [Multibase](https://github.com/multiformats/multibase): Self-identifying base encodings.
+    - [Multicodec](https://github.com/multiformats/multicodec): Compact, self-describing codecs.
+    - [Multihash](https://github.com/multiformats/multihash): Self-describing hashes for future-proofing.
+
+### IPFS tools and products
+
+- [IPFS Cluster](https://github.com/ipfs/ipfs-cluster): Automatically allocate, replicate, and track your data as a global pinset distributed among a swarm of peers.
+- [IPFS Companion](https://github.com/ipfs/ipfs-companion): The IPFS web browser extension.
+- [IPFS Web UI](https://github.com/ipfs/ipfs-webui): An easy-to-use web interface for IPFS nodes.
+- [IPFS Desktop](https://github.com/ipfs/ipfs-desktop): Standalone IPFS app with an easy-to-use node interface, plus menubar/tray shortcuts
+- [ipfs-gui](https://github.com/ipfs/ipfs-gui): Coordinating development, user experience, and maintenance of IPFS GUIs.
+- [i18n](https://github.com/ipfs-shipyard/i18n): The IPFS Translation Project, crowdsourcing translations of IPFS GUIs and websites.
+- [IPFS Shipyard](https://github.com/ipfs-shipyard): GitHub org showcasing incubated projects of all types created by the global IPFS community.
+- [Testground](https://github.com/testground/testground): A platform for testing, benchmarking, and simulating distributed and p2p systems at scale.
+
+### Interoperability, platforms, and standards
+- [IPFS Web Browsers Integration Group](https://github.com/ipfs/in-web-browsers): Tracking progress toward native IPFS support in web browsers.
+- [Are We Distributed Yet?](https://github.com/arewedistributedyet/arewedistributedyet): Prioritized listing of progress toward making peer-to-peer a first-class part of the web.
+- [IPFS Mobile Design Guidelines](https://github.com/ipfs/mobile-design-guidelines): Best practices for making IPFS work for mobile.
+- [Interoperability Tests for IPFS](https://github.com/ipfs/interop): On-the-wire interop for IPFS.
+
+## Participate in IPFS project operations
+
+IPFS project operations at large are also captured in GitHub. These repos don't necessarily contain code but do follow a similar pattern of issue creation, discussion, and resolution via comments and linked artifacts.
+
+- [IPFS Community](https://github.com/ipfs/community)
+- [IPFS Project Roadmap](https://github.com/ipfs/roadmap)
+- [IPFS Specifications](https://github.com/ipfs/specs)
+- [IPFS Infrastructure](https://github.com/ipfs/infra)
+- [IPFS Team Planning, Management, and Coordination](https://github.com/ipfs/team-mgmt)
+
+## Discuss ideas for using IPFS
```

Comment:
Can you add a link to the ipfs/community discussions board to this section? it's also a place where people are using IPFS and talking about it

https://github.com/ipfs/community/discussions